src/pages/certifications/javascript-algorithms-and-data-structures/basic-javascript/testing-objects-for-properties/index.md

```diff
@@ -3,8 +3,21 @@ title: Testing Objects for Properties
 ---
 ## Testing Objects for Properties
```

Comment:
Please use the following formatting for the `title` of a article.

```
---
title: Testing Objects for Properties
---
```
Also, please refer to our [Style Guide](https://github.com/freeCodeCamp/guide/blob/master/CONTRIBUTING.md#article-style-guide) to make sure that the rest of article is using all the correct styling.